.github/ISSUE_TEMPLATE/task.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: Task
-about: Template for a individual task or a task under an Epic
+about: Template for an individual task or a task under an Epic
 title: '<Short actionable title>'
-labels: needs discussion
```

Comment:
We could add `needs triage` label if you think that would be helpful

Reply:
I'd prefer not to. IMO the default state is "needs triage", so I think removing the label altogether is good.